index.html

```diff
@@ -19,5 +20,11 @@ <h2>Score</h2>
     <div id="memory-board"></div>
 
     <!-- LINK THE JAVASCRIPT FILES HERE (keep in mind that the order in which you link them MATTERS) -->
```

Comment:
```<link rel=“stylesheet” href=“styles/style.css”/>
<script src=“js/memory.js”></script>
<script src=“js/index.js”></script>```

Reply:
Algunos métodos de array mutan el mismo, por lo que necesitas crear una copia del mismo que le pasas como argumento, pero no siempre es necesario.

No es necesario para hacer uso de la palabra clave _this_ estar dentro del contexto de una arrow function, te dejo [link](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/this) a la documentación de MDN para que te aclares un poco, sería interesante que preguntases en clase por esta cuestión.

Tranquila porque es absolutamente normal tener esa sensación en determinadas ocasiones a lo largo del bootcamp, en tu caso en particular no tienes de que preocuparte, buen trabajo y ánimo!